### ISSUE TITLE
[BUG] Line is rendered behind fill in Chart

### ISSUE THREAD

**Issue by MadsBuchmann** [kirbydesign/designsystem#1906]
- [X] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the bug
When applying a fill to one of the datasets in chart, lines will be rendered behind that fill as can be seen on the below image:
![image](https://user-images.githubusercontent.com/17269698/143035396-3e3b67ab-4efc-48ea-b661-4979c7334cb4.png)

(It might be a tad hard to see but the green line in the middle is darker than it would otherwise be if it was not behind the fill). 

## Which Kirby version was used?
4.0.20

## What was the expected behavior?
The line should be rendered in front of the fill by default.

## Are there any additional context?
This was reported to me by @bdytos - and we played around with different ways of solving it in the consumer application. But none of them worked. 

1. We tried setting the draw time via custom options: 
```typescript 
_customOptions: ChartOptions = {
   plugins: {
      filler: {
         drawTime: 'beforeDatasetDraw'
      }
   }
}
```

2. Rearranging datasets to see if this affected rendering 
3. Using the order [option](https://www.chartjs.org/docs/latest/charts/line.html#general) to tell chart.js to draw the line on top. 

![image](https://user-images.githubusercontent.com/17269698/143043068-947e1b89-dca7-4ac5-8c6f-18c75ab5c8c2.png)

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/master/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Verification
To make sure the bug is not intended behaviour; it should be verified by a member of team Kirby before moving on to implementation. 

- [ ] Make sure the `NOT verified` label has been removed by a member of [team Kirby](https://github.com/kirbydesign/designsystem/blob/master/.github/SUPPORT.md#team-kirby); do not proceed until this is done

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/master/.github/CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [master branch](https://github.com/kirbydesign/designsystem/tree/master) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Create a test that reproduces the bug following guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)". 
- [ ] Write code to fix the bug

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request.

**Comment by RasmusKjeldgaard** [comment]
Closing this issue automatically because it has been open for more than 6 months and is not currently in progress. If this is still relevant, please feel free to reopen it or open a new issue with updated details. Thank you!
